assets/src/edit-story/components/helpCenter/useHelpCenter/test/useHelpCenter.js

```diff
@@ -268,16 +136,9 @@ describe('useHelpCenter', () => {
       expect(result.current.state.readTips).toStrictEqual(expected);
 
       // go through the list backwards
-      for (
-        let i = result.current.state.navigationFlow.length - 2;
-        i >= 0;
-        i--
-      ) {
+      for (let i = lastTipIndex; i >= 0; i--) {
```

Comment:
navigation flow changes when we move to this tip, so `result.current.state.navigationFlow.length` isn't accurate since it now contains:
```
[
  ...BASE_NAVIGATION_FLOW,
  ...unreadTipsBeforeCurrentIndex,
  doneTip
]
```